modules/core/misc/java/gen_dict.json

```diff
@@ -615,39 +615,64 @@
             "jn_args": [
                 [
                     "double",
-                    ".val[0]"
```

Comment:
I propose to presume `.val` here and below for compatibility with existing code.